src/components/common/TmSessionExplore.vue

```diff
@@ -97,7 +97,7 @@ import { formatAddress } from "src/filters"
 import { isAddress } from "web3-utils"
 const isEthereumAddress = isAddress
 const isPolkadotAddress = address => {
-  const polkadotRegexp = /[0-9a-zA-Z]{48}/
+  const polkadotRegexp = /[0-9a-zA-Z]{47}/
```

Comment:
o.O mine is 48 characters long. @mariopino is there a difference in the address formats?

Reply:
In Polkastats we have even varchar(47) in the DB and everything works perfect

So weird